pkg/assets/handlers.go

```diff
@@ -141,6 +144,18 @@ func HTML5ModeHandler(contextRoot string, subcontextMap map[string]string, h htt
 	}), nil
 }
 
+var versionTemplate = template.Must(template.New("webConsoleVersion").Parse(`
+window.OPENSHIFT_VERSION = {
+  openshift: "{{ .OpenShiftVersionInfo.GitVersion | js}}",
+  kubernetes: "{{ .KubernetesVersionInfo.GitVersion | js}}"
+};
+`))
```

Comment:
Yeah, I had it originally and removed for cleanliness, but having similar environments is a good argument. Fixed.